src/transformers/training_args.py

```diff
@@ -1550,6 +1510,32 @@ class TrainingArguments:
         },
     )
 
+    # DEPRECATED arguments, remove in 5.0.0
```

Comment:
Isn't this kinda soon? Maybe keep the deprecations a bit longer?

Reply:
cc @ArthurZucker @SunMarc if you have thoughts on a safe deprecation